### DISCUSSION TITLE
211206 rMD dev Meeting Protocol

### DISCUSSION THREAD

**Discussion by ehhartmann** [graeter-group/kimmdy#22] [Ideas]
- write sequence of tasks in yaml file with single and recurring tasks, the task queue is built from that
- recurring tasks that are ordered in a recurring tasks group have a mult variable associated with the group that denotes how often the group of tasks is repeated
- a task is allowed to build an inner task queue if subsequent tasks are necessary
- a global step variable with be used to count and limit the overall number of tasks (including inner tasks)
- no other counting variables should be used as of now, this may change later
- functions should have a documentation string
- the reaction recipe should be attached to all generated rates
- atomtypes should not be part of the reaction recipe
- reactions is a ABC and should not contain functions that are useful for all reaction classes, these can be stored in the utils file

**Comment by jmbuhr** [comment]
For future reference: All current examples are run with a plumed-patched gromacs 2021 version compiled by @ehhartmann which can be enabled by `source /hits/fast/mbm/hartmaec/sw/gromacs-2021/run/bin/GMXRC`
